apps/channel/queries/channelCan.js

```diff
@@ -4,6 +4,15 @@ export default gql`
   query channel_can($id: ID!) {
     channel(id: $id) {
       id
+      owner {
+        __typename
+        ... on Group {
+          id
+        }
+        ... on User {
+          id
+        }
+      }
```

Comment:
So this isn't related to channelCan and shouldn't be in here. Could just make this it's own fragment.

Reply:
OR—just rename this since it's legacy

Reply:
legacyChannelData or somethin

Reply:
I shall